docs/tests/approvals-tests.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Approvals Tests
+---
+
+# Approvals Tests
+
+Approvals are based on the idea of the golden master. You take a snapshot of an
```

Comment:
I feel like the sentence 'You take a snapshot' is not very articulate here. I would be more specific if possible and replace you with specificity.

Reply:
I quite literally took this from their [official repo](https://github.com/kytrinyx/approvals). Yeah, I agree that pluralizing it sounds a bit off but I'm just keeping it consistent with the gem name.